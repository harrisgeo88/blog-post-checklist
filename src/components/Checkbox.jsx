import React from 'react'

export const Checkbox = ({ title }) => {
  return (
    <label className="checkbox">
      <input type="checkbox" />
      <span className="checkbox-title">{title}</span>
    </label>
  )
}
