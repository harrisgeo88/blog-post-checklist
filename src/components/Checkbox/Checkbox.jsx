import React from 'react'

export const Checkbox = ({ title }) => {
  return (
    <label className="checkbox">
      <input type="checkbox" />
      <span className="checkbox-title" data-testid="checkbox">
        {title}
      </span>
    </label>
  )
}
