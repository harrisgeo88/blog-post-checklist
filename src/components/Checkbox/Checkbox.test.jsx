import React from 'react'
import { render } from '@testing-library/react'
import { Checkbox } from './Checkbox'

test('Checkbox matches snapshot', () => {
  const { container } = render(<Checkbox title="i like cookies" />)
  expect(container.firstChild).toMatchSnapshot()
})
