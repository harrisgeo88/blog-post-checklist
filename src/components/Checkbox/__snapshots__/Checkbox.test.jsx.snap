// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox matches snapshot 1`] = `
<label
  class="checkbox"
>
  <input
    type="checkbox"
  />
  <span
    class="checkbox-title"
    data-testid="checkbox"
  >
    i like cookies
  </span>
</label>
`;
